vendor/github.com/opencontainers/selinux/go-selinux/selinux.go

```diff
@@ -498,14 +580,14 @@ func DupSecOpt(src string) []string {
 		con["level"] == "" {
 		return nil
 	}
-	return []string{"label=user:" + con["user"],
-		"label=role:" + con["role"],
```

Comment:
This change seems like it's going to break stuff.
Why the change?

Definitely breaking https://github.com/moby/moby/blob/master/daemon/cluster/executor/container/container.go#L624